enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -55,27 +57,29 @@ const _StaticDashboard = ({
     });
 
   const { hasNightModeToggle, isNightMode, onNightModeChange, theme } =
-    useEmbedTheme(options.theme);
+    useEmbedTheme();
+
+  const { font } = useEmbedFont();
 
   return (
     <Box ref={ref} style={{ overflow: "auto" }}>
       <PublicOrEmbeddedDashboard
         dashboardId={dashboardId}
         parameterQueryParams={parameterQueryParams}
-        bordered={options.bordered}
-        font={options.font}
         hasNightModeToggle={hasNightModeToggle}
         hideDownloadButton={options.hideDownloadButton}
         hideParameters={options.hideParameters}
         isNightMode={isNightMode}
         onNightModeChange={onNightModeChange}
-        theme={theme}
         titled={options.titled}
+        theme={theme}
         isFullscreen={isFullscreen}
         onFullscreenChange={onFullscreenChange}
         refreshPeriod={refreshPeriod}
         onRefreshPeriodChange={onRefreshPeriodChange}
         setRefreshElapsedHook={setRefreshElapsedHook}
+        font={font}
+        bordered={options.bordered}
```

Comment:
Is `options.bordered` here a fixed value in this case? Since we don't accept any props for `bordered`